src/Raven.Server/Documents/Replication/DocumentInfoHelper.cs

```diff
@@ -24,6 +24,22 @@ public unsafe LazyStringValue GetDocumentId(Slice key)
             _tmpLazyStringInstance.Renew(null, key.Content.Ptr, sepIdx, _context);
             return _tmpLazyStringInstance;
         }
+        
+        public LazyStringValue GetDocumentId(ReplicationBatchItem item)
```

Comment:
That is code that was moved, no?

Reply:
not sure I follow.. this is a new code.

Reply:
This is very similar to code that I'm familiar with. Is this code duplication?

Reply:
indeed :)
similar to the `ShouldAllow` from the `AllowedPathsValidator` but serve different purpose.
Maybe eventually they will be unified.

Reply:
Okay, make a note to clean it later, then

Reply:
done